src/docs/product/issues/states-triage/index.mdx

```diff
@@ -0,0 +1,85 @@
+---
+title: Issue States and Triage
+sidebar_order: 30
+description: "Learn more about issue states and how to triage issues."
+---
+
+To help organize issues for you and your team, you can take actions such as assigning, resolving, or ignoring issues. These actions are referred to as _triaging_.
+
+An issue can be in one of three states:
+
+1. Unresolved: The default state for all new issues.
+2. Ignored: The issue is ignored permanently or until a condition is met to move it to a state of Unignored. [Issue alerts](/product/alerts-notifications/issue-alerts/) are suppressed for ignored issues.
+3. Resolved: The issue is resolved because it was fixed. You can also mark an issue as Resolved if you don't expect it to happen again.
+
+## Issue Triage
+
+Triaging is the act of deciding what action you want to take on an issue. It typically involves one or more of these actions.
+
+### Assign
+
+You can assign issues to teams or individuals manually or let issues be automatically assign by setting up [ownership rules](/product/issues/issue-owners/). This helps you and your team see only the issues that you care about. You might also see a list of suggested assignees including the owners of suspect commits. Typically, assigning an issue to a team indicates that the team owns the issue and that they're responsible for triaging it further.
```

Comment:
"This allows you to route issues to the right people and helps you and your team see only relevant issues."